sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -120,3 +120,73 @@ public ProvisionableTemplateAttribute(string resourceName) { }
         public string ResourceName { get { throw null; } }
     }
 }
+namespace System.ClientModel
+{
+    public partial interface IAuthorizationCodeFlowContext : System.ClientModel.IScopedFlowContext, System.ClientModel.ITokenContext
```

Comment:
Do all of these need to be interfaces? Can we change them to abstract classes?

Reply:
The thinking is that these flows need not be mutually exclusive in an TokenProvider implementation type. It would be more complicated to do that with class inheritance.